src/commands/appdistribution-distribute.ts

```diff
@@ -43,13 +50,39 @@ export const command = new Command("appdistribution:distribute <release-binary-f
     "--groups-file <file>",
     "path to file with a comma separated list of group aliases to distribute to",
   )
+  .option(
+    "--test-devices <string>",
+    "semicolon-separated list of devices to run automated tests on, in the format 'model=<model-id>,version=<os-version-id>,locale=<locale>,orientation=<orientation>'. Run 'gcloud firebase test android|ios models list' to see available devices",
```

Comment:
can you add a note in here about this feature being in beta to keep the language in line with our other CLI tools?

Reply:
Done! I added all the similar updates from the fastlane plugin: https://github.com/firebase/fastlane-plugin-firebase_app_distribution/commit/e14eb83330b14f20bd7bc6214429e6cb9e74dc47